diff --git a/analyzer/tools/build-logger/Makefile.manual b/analyzer/tools/build-logger/Makefile.manual
index 351a0f05..5f16ed16 100644
--- a/analyzer/tools/build-logger/Makefile.manual
+++ b/analyzer/tools/build-logger/Makefile.manual
@@ -68,7 +68,7 @@ pack64bit: 64bit packbin
 # pack binary
 packbin:
 	mkdir -p $(BIN_DIR)
-	cp ldlogger $(BIN_DIR)
+	cp ldlogger $(BIN_DIR) || true
 	rm -f ldlogger
 
 # Build only 32 bit lib
